libs/extensions/src/settings/settings.service.ts

```diff
@@ -103,79 +136,119 @@ export class SettingsService {
     SettingsService.data = result;
   }
 
-  async update(name: string, value: any) {
-    // Ensure that the name is stored in uppercase
-    const uppercaseName = name.toUpperCase();
-
-    // Check if the setting exists in the database
-    const existingSetting = await this.prisma.setting.findUnique({
-      where: { name: uppercaseName },
+  async update(name: string, dto: UpdateSettngsDto) {
```

Comment:
Please get rid of the comment out portion

Reply:
@santosh-rumsan, unnecessary comment portion removed